packages/forma-36-website/src/pages/components/grid/index.mdx

```diff
@@ -6,4 +6,23 @@ github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-re
 storybook: 'https://f36-storybook.contentful.com/?path=/story/components-grid--default'
 ---
 
-Detailed documentation will be provided soon. For now, please have a look into the storybook for implementation details.
\ No newline at end of file
+Detailed documentation will be provided soon. For now, please have a look into the storybook for implementation details.
+
+```jsx
+<Grid rows={2} rowGap="spacingM">
+  <Grid columns={5}>
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+  </Grid>
+  <Grid columns={5}>
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+    <GridItem style={{backgroundColor: '#000', width: '150px', height: '100px'}} />
+  </Grid>
+</Grid>
+```
```

Comment:
![Screenshot 2020-06-11 at 16 53 46](https://user-images.githubusercontent.com/6597467/84402079-4f63ab00-ac04-11ea-93f9-889a25595edc.png)

Is this the problem in the alpha components you were mentioning?

Reply:
@gui-santos w8, I will fix it

Reply:
It would work now, but.... eslintrc again. (I understood myself now, reverse engineering of my own thing. 🤦‍♀️). Maybe you will figure out how to solve it somehow differently

Reply:
Maybe there is a `: any` in the Grid component... I will see if I can help :)

Reply:
@gui-santos we can try to solve it in a separate PR. Maybe we could merge this one and then when fix is ready we will remove `.eslintrc`? or better to solve it right away?

Reply:
I guess we can fix later

But just to update you this is what I found. I removed the .eslintrc and everything worked fine
![Screenshot 2020-06-11 at 17 40 13](https://user-images.githubusercontent.com/6597467/84407063-a0769d80-ac0a-11ea-921a-9bbdfce421e7.png)

But I'm getting a bunch of lint warnings (not errors) in the /dist folder, which I think should not happen because we should not lint the builds, right?

Reply:
This is one of the warnings, see the **dist/components/Grid/Grid.js**

```
/Users/.../forma-36/packages/forma-36-react-components/dist/components/Grid/Grid.js
   1:1    warning  'use strict' is unnecessary inside of modules  strict
  26:185  warning  Don't make functions within a loop             no-loop-func
  26:429  warning  Don't make functions within a loop             no-loop-func
```

Reply:
And for me, if I remove `.eslintrc` I get this 🙁

<img width="1185" alt="Screenshot 2020-06-11 at 16 57 42" src="https://user-images.githubusercontent.com/4272331/84409510-d0737000-ac0d-11ea-8bf1-50e1ddf9d075.png">

Reply:
Let's fix this voodoo in another PR!